frontends/mit-learn/src/pages/HomePage/HomePage.test.tsx

```diff
@@ -315,55 +318,12 @@ describe("Home Page Testimonials", () => {
 })
 
 describe("Home Page Carousel", () => {
-  test.each<{ tab: string; params: FeaturedRequest }>([
-    {
-      tab: "All",
-      params: { limit: 12, resource_type: ["course"] },
-    },
-    {
-      tab: "Free",
-      params: { limit: 12, resource_type: ["course"], free: true },
-    },
-    {
-      tab: "With Certificate",
-      params: {
-        resource_type: ["course"],
-        limit: 12,
-        certification: true,
-        professional: false,
-      },
-    },
-    {
-      tab: "Professional & Executive Learning",
-      params: { resource_type: ["course"], limit: 12, professional: true },
-    },
-  ])("Featured Courses Carousel Tabs", async ({ tab, params }) => {
-    const resources = learningResources.resources({ count: 12 })
-    setupAPIs()
-
-    // The tab buttons eager-load the resources so we need to set them all up.
-
-    // This is for the clicked tab (which might be "All")
-    // We will check that its response is visible as cards.
-    setMockResponse.get(
-      urls.learningResources.featured({ ...params }),
-      resources,
-    )
-
-    renderWithProviders(<HomePage />)
-    screen.findByRole("tab", { name: tab }).then(async (featuredTab) => {
-      await user.click(within(featuredTab).getByRole("tab", { name: tab }))
-      const [featuredPanel] = screen.getAllByRole("tabpanel")
-      await within(featuredPanel).findByText(resources.results[0].title)
-    })
```

Comment:
I decided to just remove this test. It was broken and not adding much value, IMO. Testing the behavior of the resource carousel (clicking tabs, etc) is pretty well covered in ResourceCarousel.test.tsx

**How it was broken:** `screen.findByRole("tab", ...` at the end here is a promise and it was not being awaited.
- if you insert an `await` here you'll see the test fails on `main`
- Usually un-awaited asynchronous assertions can cause artificial failures on **subsequent** async tests in the same file, but since this was the last test with any `await`s, we weren't noticing any.